Tupfile

```diff
@@ -21,10 +21,6 @@ else
 srcs += src/sqlite3/*.o
 endif
 ifneq ($(TARGET),win32)
-ifeq (@(TUP_SUDO_SUID),y)
```

Comment:
There are still some systems that don't have namespacing support by default, so having suid support makes things easier I think. In any case, I don't think this has to do with Windows support, so it should probably be a separate pull request?